### PR TITLE
feat(visibility-utils): add responsive utility classes for visibility

### PR DESCRIPTION
**Pull request by zak-eddingtonPOP**
Add responsive utility classes and mixins for controlling visibility of elements.

This provides a standard way of showing/hiding content across breakpoints without needing to adding custom styles per component. It also adds a screen reader only style for accessibility.

### CODE REVIEW

File: src/scss/mixins/visibility.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	text-indent: 200%;
+	white-space: nowrap;
+	width: 1px;
+}
+
+@mixin hidden {
+	display: none !important;
```
Comment (brian-patrick-3) [inline src/scss/mixins/visibility.scss:15]: Is the `!important` necessary?

**Review by brian-patrick-3 — CHANGES_REQUESTED** [review]
We should document these new utility classes somewhere in Storybook, otherwise no one will know they exist.

Also, please consider running Prettier on your files, I see some strange indentation. The Prettier pre-commit hook is supposed to catch any formatting discrepancies, but it seems it hasn't here.

File: src/scss/mixins/visibility.scss (hunk 1 above)
Comment (zak-eddingtonPOP) [reply src/scss/mixins/visibility.scss]: Yeah, this is to prevent being overridden by more specific selectors or the order of styles.

**Comment by zak-eddingtonPOP** [comment]
I've added a documentation page, and converted my tabs to spaces to match indentation being used elsewhere.

**Review by brian-patrick-3 — APPROVED** [review]

**Pull request merged**